install-ci.yml

```diff
@@ -60,6 +69,14 @@
             - daily
             - notifempty
 
+    - role: filebeat
```

Comment:
We'll need to set 'filebeat_output_logstash_hosts' to point at the monitoring host, which will be different in each ansible environment (production, vagrant, docker, and the multinode test)